ftw/solr/handlers.py

```diff
@@ -85,7 +85,7 @@ def get_data(self, attributes):
         # in Solr. Because path_depth is not a catalog index, we would
         # otherwise fail to update it in cases where a specific list of
         # `idxs` including path is passed to reindexObject().
-        if 'path' in attributes:
+        if 'path' in attributes and 'path_depth' in schema.fields.keys():
```

Comment:
Use `path_depth in schema.fields` instead of `path_depth in schema.fields.keys()` because it's more efficient. Time complexity: O(1) vs. O(n)